app/controllers/sail/settings_controller.rb

```diff
@@ -14,6 +14,12 @@ def update
       end
     end
 
+    def show
```

Comment:
Similarly to the update action, can you please envelop this in a respond_to block with JSON? Something like this

```ruby
def show
  respond_to do |format|
    format.json do
        setting = Sail::Setting.get(params[:name])
        render json: { value: setting }
    end
  end
end
```

Reply:
Fixed :)